### DISCUSSION TITLE
Osu doesn't start

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#21535] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

I accidentaly pressed esc button to much - osu was closed before update was downloaded. Now I can't start the game. Opening exe file has no effect - nothing happening. 
Windows events log: 
[osu.zip](https://github.com/ppy/osu/files/10168950/osu.zip)

### Version

Previous (I can't see)

### Logs

[performance.log](https://github.com/ppy/osu/files/10168785/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10168787/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10168788/updater.log)
[database.log](https://github.com/ppy/osu/files/10168789/database.log)
[input.log](https://github.com/ppy/osu/files/10168790/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10168791/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10168792/network.log)

**Comment by bdach** [comment, marked as answer]
The logs attached look like the game is running perfectly fine. Which means that the app presumably isn't even starting.

Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Reply by okolenmi** [reply]
[osu.zip](https://github.com/ppy/osu/files/10168949/osu.zip)

**Reply by okolenmi** [reply]
Oh, I see there are directories with previous versions. The current version size is only 60MB when other two 200+MB each. I deleted new folder and package. Now osu works.

**Reply by okolenmi** [reply]
I hope that developers will remove an ability to run not fully dowloaded versions.

**Reply by peppy** [reply]
Do you still have a copy of the incomplete update? If so I'd be very curious to see the state it was in.
